### PR TITLE
Big rework (no lazy require, CJS => ESM sources, rollup to CJS build)

### PR DESCRIPTION
**Pull request by hsablonniere**
This PR includes several major refactors and some late changes fixes for this refactor:

* no lazy require #773
* CJS => ESM sources #771
* rollup to CJS build #807
* Update to new clever-client with breaking change on crypto #830
* Use @clevercloud/client esm instead of cjs #831
* Fix for database default command not working post rework #829
* Use `node:` for Node.js modules

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  output: {
+    file: 'build/clever.cjs',
+    format: 'cjs',
+    sourcemap: 'inline',
+  },
+  plugins: [
+    {
+      transform (code, id) {
+
+        // formiable (used by superagent) hijacks require :-(
```
Comment (miton18) [inline rollup.config.js:18]: typo ?

File: src/load-package-json.cjs
Comment (miton18) [inline src/load-package-json.cjs:1]: maybe later https://nodejs.org/dist/v18.15.0/docs/api/esm.html#esm_import_assertions

**Review by miton18 — CHANGES_REQUESTED** [review]

**Comment by miton18** [comment]
QA: Works on my side

File: rollup.config.js (hunk 1 above)
Comment (hsablonniere) [reply rollup.config.js:18]: Good catch

File: src/load-package-json.cjs
Comment (hsablonniere) [reply src/load-package-json.cjs:1]: Yes, except the syntax and name of the feature has changed, it's now the `with` keyword and Node unshipped the `assert` syntax.

https://nodejs.org/api/esm.html#import-attributes

We'll be able to go for v22 soon.

**Pull request merged**